xblock/core.py

```diff
@@ -174,6 +174,15 @@ def validate(self):
         """
         return Validation(self.scope_ids.usage_id)
 
+    def ugettext(self, text):
```

Comment:
Hmm, not sure I understand -- I think we should keep the name of this function "ugettext" as advised by @nedbat.  In the individual XBlocks we might think about setting a module-level variable "_" as a reference to "self.ugettext" for marking up the actual strings -- is that what you mean?